_layouts/contact.html

```diff
@@ -23,18 +21,18 @@ <h1>Contact Us</h1>
         <div class="row">
             <div class="col-md-9 col-xm-12 block-center">
                 <div class="row">
-                    <div class="col-sm-4 col-xs-12 mailing-lists">
+                    <div class="col-sm-4 col-12 mailing-lists">
                         <h2 class="uppercase">Mailing Lists</h2>
```

Comment:
I think we can safely remove that section; the list is practically dead even though we never officially dealt the killing blow (ie announcing it on the list and actually disabling new posts)